packages/teams-js/src/public/authentication.ts

```diff
@@ -28,39 +33,53 @@ export namespace authentication {
   let authParams: AuthenticateParameters | undefined;
   /**
    * @deprecated
-   * As of 2.0.0, this function has been deprecated in favor of a Promise-based pattern.
-   * Registers the authentication Communication.handlers
+   * As of 2.0.0, this function has been deprecated in favor of a Promise-based pattern using {@link authentication.authenticate authentication.authenticate(authenticateParameters: AuthenticatePopUpParameters): Promise\<string\>}
+   *
+   * Registers handlers to be called with the result of an authentication flow triggered using {@link authentication.authenticate authentication.authenticate(authenticateParameters?: AuthenticateParameters): void}
    *
-   * @param authenticateParameters - A set of values that configure the authentication pop-up.
+   * @param authenticateParameters - Configuration for authentication flow pop-up result communication
    */
   export function registerAuthenticationHandlers(authenticateParameters: AuthenticateParameters): void {
     authParams = authenticateParameters;
   }
 
   /**
-   * Initiates an authentication request, which opens a new window with the specified settings.
+   * Initiates an authentication flow which requires a new window.
+   * There are two primary uses for this function:
+   * 1. When your app needs to authenticate using a 3rd-party identity provider (not Azure Active Directory)
+   * 2. When your app needs to show authentication UI that is blocked from being shown in an iframe (e.g., Azure Active Directory consent prompts)
+   *
+   * You can learn more details about this flow [here](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/authentication/auth-flow-tab)
```

Comment:
How about: 

`For more details, see [Enable authentication using third-party OAuth provider](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/authentication/auth-flow-tab).`